keps/sig-architecture/2019-03-19-go-modules.md

```diff
@@ -166,9 +213,14 @@ This section is in progress, pending investigation into necessity and feasibilit
 
 ### Graduation Criteria
 
-* kubernetes/kubernetes vendor management uses go modules
-* published staging components can be successfully consumed by go module consumers
-* published staging components can be successfully consumed by GOPATH-based consumers
+* `k8s.io/kubernetes` vendor management uses go modules
```

Comment:
I would propose graduation include "CI checks all build and run outside of a GOPATH"

Reply:
> I would propose graduation include "CI checks all build and run outside of a GOPATH"

I want to be careful to keep the `godeps -> go modules` work scoped so that it can be completed and "graduate" in 1.15.

Changing build scripts from `GOPATH -> go modules` is a distinct effort (and one we should do), but is not required to get our vendor management and staging component consumption ready for go 1.13. I'm not necessarily opposed to including it in this doc, but I think that muddies the water about when each effort is completed.